mw-webapp/src/component/select/Select.tsx

```diff
@@ -0,0 +1,53 @@
+import {Option} from "src/component/select/option/Option";
+import {OptionType} from "src/component/select/option/Option";
+import styles from "src/component/select/Select.module.scss";
+
+
+/**
+ * Select component
+ */
+export interface SelectProps {
+  /**
+   * Label's text
+   */
+  label: string;
+  /**
+   * Default select value
+   */
+  value: string;
+  /**
+   * Select's name
+   */
+  name: string;
+  /**
+   * Options list
+   */
+  options: OptionType[];
+  /**
+   * Default option
+   */
+  onChange: (e: React.ChangeEvent<HTMLSelectElement>) => void;
```

Comment:
Why there is "Default option", it's a callback

Reply:
done